ui/round/src/clock/view.js

```diff
@@ -29,7 +29,13 @@ function formatClockTime(ctrl, time, running) {
 }
 
 function showBar(ctrl, time) {
-  return ctrl.data.showBar ? m('div.bar',
+  function chooseColor(seconds) {
+      if (seconds > 30) return 'green';
+      else if (seconds > 20) return 'yellow';
+      else if (seconds > 10) return 'orange';
+      else return 'red';
+  }
+  return ctrl.data.showBar ? m('div.bar.' + chooseColor(time),
```

Comment:
Light gold instead? The current gold is too catchy, it's even more alerting than red.